PageViews/PageViews.js

```diff
@@ -69,6 +67,20 @@ const PageViewsBase = kind({
 		 */
 		componentRef: EnactPropTypes.ref,
 
+		/**
+		 * Customizes the component by mapping the supplied collection of CSS class names to the
+		 * corresponding internal elements and states of this component.
+		 *
+		 * The following classes are supported:
+		 *
+		 * * `pageViews` - The root component class
+		 * * `navButton` - The navButton component class
```

Comment:
The `publicClassNames` is `true`. Are these all of the supported class names?

Reply:
It's added.